config/rbac/role.yaml

```diff
@@ -248,6 +248,23 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Do we need these permissions to the operator itself? When it interacts with these CRs?

Reply:
ack